spec/models/user_spec.rb

```diff
@@ -5,7 +5,37 @@
     @user = create :user
   end
 
-  it 'works' do
-    expect(@user.first_name).to be('Test')
+  describe '#full_name' do
+    it 'returns users full name' do
+      expect(@user.full_name).to eq('Test User')
+    end
+  end
+
+  describe '#union' do
```

Comment:
You have spaces around all your describe blocks here but not above. Which is better?

Reply:
I'm of the opinion that space between describe blocks improves readibility as the sections being tested are more visually distinct. I changed the shift spec to have more space.